plugin/externalPluginEntry.go

```diff
@@ -33,6 +36,7 @@ func unixSecondsToTimeAttr(seconds int64) time.Time {
 	return time.Unix(seconds, 0)
 }
 
+/*
```

Comment:
Why are you keeping this around?

Reply:
So I can look at it later once I get around to doing it. Don't mind removing it if that's preferred too.

Reply:
So I can look at it later once I get around to doing it. Don't mind removing it if that's preferred too.